focus-android/app/metrics.yaml

```diff
@@ -2357,6 +2360,64 @@ cookie_banner:
       tags:
         - Privacy&Security
 
+  report_site_domain:
+    type: url
+    description: |
+      A user can report a site domain when the cookie banner
+      reducer is not working from the cookie banner details panel.
```

Comment:
I recommend explaining what is meant by "domain" here. It appears to be eTLD + 1 (if I'm reading this Kotlin properly)

Reply:
Done